process_jartic_archives.py

```
@@ -0,0 +1,281 @@
+    
+    total_measurements = 0
+    processed_archives = 0
+    
+    # Process each archive
+    for i, archive_path in enumerate(archives):
+        try:
+            month_str = archive_path.stem.replace('jartic_typeB_', '').replace('_', '-')
+            current_archive.set_postfix_str(f"{month_str}")
+            
+            archive_measurements = 0
+            async for archive_index, month_str, measurements, loc_count, total_locs in process_single_archive(
```

Ruff (0.12.2)

Loop control variable `archive_index` not used within loop body

Rename unused `archive_index` to `_archive_index`

(B007)